cmd/zc_enumerator.go

```diff
@@ -48,6 +48,7 @@ type storedObject struct {
 	size             int64
 	md5              []byte
 	blobType         azblob.BlobType // will be "None" when unknown or not applicable
+	contentEncoding  string
```

Comment:
You may want to base on the s2s refactor. It already takes in parameters like this, and we may well be stepping on each other's toes. That or, whichever of our PRs gets merged in last needs to resolve this.

Reply:
Good point - I'll hold off and rebase once yours is in.

Reply:
Done. Thanks for the tip.